pymodule.py

```diff
@@ -306,8 +315,12 @@ def run_forte(name, **kwargs):
     # Create the MOSpaceInfo object
     mo_space_info = forte.make_mo_space_info(ref_wfn, forte.forte_options)
 
-    # Create the AO subspace projector
-    ps = forte.make_aosubspace_projector(ref_wfn, options)
+    # Call methods that project the orbitals (AVAS, embedding)
+    orbital_projection(ref_wfn, options)
+
+    #Apply the projector to rotate orbitals
+    avas = options.get_bool("AVAS")
+    forte.apply_avas(ref_wfn, options, ps, avas)
```

Comment:
I don't think we need `apply_avas` if the logic is also in `orbital_projection`. Seems like the `make_avas` gets called twice?